src/test/java/org/broadinstitute/dsde/consent/ontology/resources/LivenessResourceTest.java

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.dsde.consent.ontology.resources;
+
+import org.junit.Test;
+import javax.ws.rs.core.Response;
+import static org.junit.Assert.assertEquals;
+
+public class LivenessResourceTest {
+    @Test
+    public void testHealthy() {
+        LivenessResource resource = new LivenessResource();
+        Response response = resource.getStatus();
+        assertEquals(200, response.getStatus());
```

Comment:
Nit: Since the API docs and the endpoint define a "Healthy" string return, you could add an assertion to check `response.getEntity()`